### PR TITLE
task 1 complete

### PR DESCRIPTION
**Pull request by rkondratowicz-legacy**
Per the instructions, this test: 
1. Opens the page given
2. Asserts that the decedent's first name appears in the Name section
3. Scrolls down until the Guest Book is visible
4. Leaves a Guest Book entry

### CODE REVIEW

**Comment by kboers-legacy** [comment]
Looks like something's missing:

```$ npx codeceptjs run test_guestbook_test.js 
creating output directory: /Users/kboers/src/rob-onboarding/output

Could not load helper WebDriver from module './helper/WebDriver':
Required modules are not installed.

RUN: npm install --save-dev webdriverio@^5.2.2
Error: Required modules are not installed.

RUN: npm install --save-dev webdriverio@^5.2.2
    at createHelpers (/Users/kboers/src/rob-onboarding/node_modules/codeceptjs/lib/container.js:177:17)
    at Function.create (/Users/kboers/src/rob-onboarding/node_modules/codeceptjs/lib/container.js:43:25)
    at Codecept.init (/Users/kboers/src/rob-onboarding/node_modules/codeceptjs/lib/codecept.js:56:15)
    at Command.module.exports (/Users/kboers/src/rob-onboarding/node_modules/codeceptjs/lib/command/run.js:24:14)
    at Command.listener (/Users/kboers/src/rob-onboarding/node_modules/commander/index.js:315:8)
    at Command.emit (events.js:315:20)
    at Command.parseArgs (/Users/kboers/src/rob-onboarding/node_modules/commander/index.js:651:12)
    at Command.parse (/Users/kboers/src/rob-onboarding/node_modules/commander/index.js:474:21)
    at Object.<anonymous> (/Users/kboers/src/rob-onboarding/node_modules/codeceptjs/bin/codecept.js:192:9)
    at Module._compile (internal/modules/cjs/loader.js:1176:30)

Error: 
    at createHelpers (/Users/kboers/src/rob-onboarding/node_modules/codeceptjs/lib/container.js:182:13)
    at Function.create (/Users/kboers/src/rob-onboarding/node_modules/codeceptjs/lib/container.js:43:25)
    at Codecept.init (/Users/kboers/src/rob-onboarding/node_modules/codeceptjs/lib/codecept.js:56:15)
    at Command.module.exports (/Users/kboers/src/rob-onboarding/node_modules/codeceptjs/lib/command/run.js:24:14)
    at Command.listener (/Users/kboers/src/rob-onboarding/node_modules/commander/index.js:315:8)
    at Command.emit (events.js:315:20)
    at Command.parseArgs (/Users/kboers/src/rob-onboarding/node_modules/commander/index.js:651:12)
    at Command.parse (/Users/kboers/src/rob-onboarding/node_modules/commander/index.js:474:21)
    at Object.<anonymous> (/Users/kboers/src/rob-onboarding/node_modules/codeceptjs/bin/codecept.js:192:9)
    at Module._compile (internal/modules/cjs/loader.js:1176:30)```

File: test_guestbook_test.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+Scenario('test something', ({ I }) => {
+    I.amOnPage('https://www.qa-legacy.com/us/obituaries/chicagotribune/name/virginia-gruchalski-obituary?pid=196167379');
+    I.see('Virginia Grace Gruchalski', { css: 'div[data-component=NameHeadingText]' });
+    I.scrollTo({ css: 'div[data-component=GuestbookBox]' });
+    I.click({ css: 'input[name=_GuestBookForm_From]' });
+    I.fillField('_GuestBookForm_From', 'John Smith');
+    I.fillField('_GuestBookForm_Message', 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed sit amet euismod augue, sit amet facilisis massa. Nullam vehicula lacus felis, dictum volutpat enim lacinia quis.');
+    I.fillField('_GuestBookForm_Email', 'test@example.com');
+    I.click('Submit Your Message');
+});
```
Comment (kboers-legacy) [inline test_guestbook_test.js:15]: nit: our standard is to add a newline at EOF

**Comment by rkondratowicz-legacy** [comment]
I think this is because I installed the **@wdio/selenium-standalone-service** package before I ran **npx codeceptjs run**. This was an alternative option given in the CodeceptJS/WebDriver [documentation](https://codecept.io/webdriver/#what-is-selenium-webdriver) to installing Selenium Server and ChromeDriver. 

This is the error I received when I used the link in the docs to install and run [selenium-standalone](https://www.npmjs.com/package/selenium-standalone):

Error in "getDownloadStream". Could not download https://msedgedriver.azureedge.net/91.0.864.53/edgedriver_mac64.zip
See more details below:
404 https://msedgedriver.azureedge.net/91.0.864.53/edgedriver_mac64.zip
Response code 404 (Not Found)
[=====] 99%      https://chromedriver.storage.googleapis.com/91.0.4472.101/chromedriver_mac64.ziError in "getDownloadStream". Could not download https://chromedriver.storage.googleapis.com/91.0.4472.101/chromedriver_mac64.zip
See more details below:
200 https://chromedriver.storage.googleapis.com/91.0.4472.101/chromedriver_mac64.zip
read ENETDOWN
/Users/rkondratowicz/Desktop/Onboarding Training/Week 1/demo_webdriver2/node_modules/selenium-standalone/lib/check-paths-existence.js:16
              return reject(new Error('Missing ' + path));
                            ^

Error: Missing /Users/rkondratowicz/Desktop/Onboarding Training/Week 1/demo_webdriver2/node_modules/selenium-standalone/.selenium/chromedriver/latest-x64-chromedriver
    at /Users/rkondratowicz/Desktop/Onboarding Training/Week 1/demo_webdriver2/node_modules/selenium-standalone/lib/check-paths-existence.js:16:29
    at suppressedCallback (node:fs:260:5)
    at FSReqCallback.oncomplete (node:fs:185:23)

**Review by kboers-legacy — APPROVED** [review]
Nice work!!

**Pull request merged**